main.js

```diff
@@ -72,20 +72,37 @@ function displayChoice(choiceType) {
 }
 
 function saveOutfit() {
-  if (currentOutfit.title === undefined) {
-    currentOutfit.setTitle(outfitNameInput.value);
+  debugger;
+  currentOutfit.setTitle(outfitNameInput.value);
+  if (outfitList.find(element => element.id === currentOutfit.id) === undefined) {
+    saveNewOutfit();
+  } else if (outfitList.find(element => element.id === currentOutfit.id)) {
+    updateExistingOutfit();
   }
+  saveToLocalStorage();
+  resetHelper();
+}
+
+function saveNewOutfit() {
   populateCard();
   var newOutfit = new Outfit(currentOutfit);
   outfitList.push(newOutfit);
-  saveToLocalStorage(newOutfit);
-  resetHelper();
+}
+
+function updateExistingOutfit() {
+  // debugger;
```

Comment:
Here's another debugger leftover in the code, can you remove it as well please.

Reply:
Sweet! Thanks for catching that!